grakn-engine/src/main/java/ai/grakn/engine/GraknApplication.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.engine;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * Main class invoked by bash scripting
+ *
+ * @author Michele Orsi
+ *
+ */
+public class GraknApplication {
```

Comment:
If we can call this class `Grakn` it'll display better to the user when they do `ps` or `jps`.

Do you think it is easy enough to do?

Reply:
We already have a class called `Grakn` but it's in a different package so it should be okay. Just need to take care with those auto imports :)

Reply:
Ok, I am going to rename this class as "Grakn"